clientGQL.go

```diff
@@ -97,7 +91,6 @@ func (client *Client) Validate() error {
 		}
 	}
 	err := client.Query(&q, nil)
-	// TODO: we should probably use a custom OpsLevelClientError type - https://www.digitalocean.com/community/tutorials/creating-custom-errors-in-go
```

Comment:
Removed because we are fairing fine without this. If we did really want to do this, we should file a ticket since it's not a trivial thing to do that can be encompassed by a `// TODO` comment.